testsuite/MDAnalysisTests/analysis/test_dihedrals.py

```diff
@@ -71,7 +109,61 @@ def test_protein_ends(self, universe):
         with pytest.warns(UserWarning):
             rama = Ramachandran(universe.select_atoms("protein")).run()
 
+    def test_None_removal(self):
+        with pytest.warns(UserWarning):
+            u = mda.coordinates.MMTF.fetch_mmtf('19hc')
+            rama = Ramachandran(u.select_atoms("protein").residues[1:-1])
+
     def test_plot(self, universe):
         ax = Ramachandran(universe.select_atoms("resid 5-10")).run().plot()
         assert isinstance(ax, matplotlib.axes.Axes), \
             "Ramachandran.plot() did not return and Axes instance"
+
+class TestJanin(object):
+
+    @pytest.fixture()
+    def universe(self):
+        return mda.Universe(GRO, XTC)
+
+    def test_janin(self, universe):
+        janin = Janin(universe.select_atoms("protein")).run()
+        test_janin = np.load(JaninArray)
```

Comment:
make the `test_janin` a class-level fixture so that it only gets loaded once:
```python


class ....:

   @pytest.fixture
   def janin_ref_array(self):
         return np.load(JaninArray)

   def test_janin(self, universe, janin_ref_array):
         janin = Janin(universe.select_atoms("protein")).run()
         assert_almost_equal(janin.angles, janin_ref_array, 4, ...)
```